### PR TITLE
Several Updates

### PR DESCRIPTION
**Pull request by deybhayden**
Each commit is its own update, but I'll line them out.

f2fbc6b:

> This allows OTP releases (ex. distillery releases) to use this library.

> From jose's readme: https://github.com/potatosalad/erlang-jose#installation

> If you are using deployment tools (exrm, etc.) and your app depends on jose directly, you will need to include jose in your applications list in mix.exs to ensure they get compiled into your release

95b58a6:

> Able to send SSML back in an Activity

9e92475:

> Add name to Entity model so can match on `Microsoft.Launch` and do different logic on app launch

3bf3662:

> Fix warnings from mix compile, update to 1.5 elixir

3f6e683:

> Add inputHint to Activity

### CODE REVIEW

**Comment by zabirauf** [comment]
Thanks @deybhayden for the PR. I'll take a look at it.

File: lib/client.ex [hunk 1]
```diff
@@ -8,7 +8,12 @@ defmodule ExMicrosoftBot.Client do
   @type error_type :: {:error, integer, String.t}
 
   def deserialize_response(%HTTPotion.Response{status_code: 200, body: body}, deserialize_func) do
-    {:ok, deserialize_func.(body)}
+    case body do
```
Comment (zabirauf) [inline lib/client.ex]: Why do we need the change here. Which decode error are you running into?
As I don't expect the response for 200 to contain empty body, if we are seeing this then I believe we should return `{:error, ...}`.

Comment (deybhayden) [reply lib/client.ex]: It was throwing a Poison decode error when running the [delete_user_data](https://github.com/zabirauf/ex_microsoftbot/blob/master/lib/botstate_client.ex#L15) function.

Comment (deybhayden) [reply lib/client.ex]: Can be removed though, if you want, I wasn't able to use the bot state API in my project after all, it didn't work for Skype or Cortana.

**Comment by deybhayden** [comment]
So, it's been awhile 😅

**Comment by zabirauf** [comment]
Hey @deybhayden, I’ll look at it very soon 🙂. Thanks for waiting.

**Comment by deybhayden** [comment]
Hey there again 😅  - I don't want to keep spamming you - but was curious if you were interested in maintaining this library any more? If not, the [StatMuse](https://github.com/statmuse/) org might be interested in taking over. I know the commit list and change is getting pretty long now.

**Comment by zabirauf** [comment]
@deybhayden I've merged it. I'll also made you a collaborator as you've been actively working on it.

**Comment by deybhayden** [comment]
Wonderful, thanks so much! 🙌

**Comment by deybhayden** [comment]
@zabirauf One more question - would you mind to add me to the Hex owners so I can publish a new version on hex.pm? Here's a link on how to do so - https://medium.com/@kdisneur/team-collaboration-on-hex-packages-70db5170d6e5

**Comment by zabirauf** [comment]
@deybhayden Done :)

**Comment by deybhayden** [comment]
Awesome, thanks! A new version has been published :tada:

**Pull request merged**